src/index.js

```diff
@@ -0,0 +1,124 @@
+function setUp(){
+    const $update = document.querySelector('#update');
+    $update.onclick = updateRates;
+
+    getFxRates();
+    setDate();
+}
+
+function updateRates(){
+    removeRates();
+    const [base, date] = readBaseAndDate();
```

Comment:
le pusiste un number espectacular a la función `readBaseAndDate`, el nombre es 100% correcto porque te está indicando que está haciendo mucho, siempre que te veas en la posición de tener que ponerle "and" a la función es que la función violó el principio de responsabilidad única. En este caso te conviene hacer 2 funciones, readBase y readRate, y listola 💯